histomicstk/utils/poisson_mixture.py

```diff
@@ -2,7 +2,7 @@
 import scipy as sp
 
 
-def PoissonMixture(I, Mu=None, InitialTau=None, Tol=0.1):
+def poisson_mixture(I, Mu=None, InitialTau=None, Tol=0.1):
```

Comment:
I think this will get refactored again as we add more mixture modeling capabilities - we can deal with this then?